strawberryfields/apps/qchem/vibronic.py

```diff
@@ -138,6 +141,42 @@ def energies(samples: list, w: np.ndarray, wp: np.ndarray) -> Union[list, float]
     return [np.dot(s[: len(s) // 2], wp) - np.dot(s[len(s) // 2 :], w) for s in samples]
 
 
+def VibronicTransition(U1: np.ndarray, r: np.ndarray, U2: np.ndarray, alpha: np.ndarray):
+    r"""An ``sf`` operation for applying the Doktorov operator
```

Comment:
I am not a big fan of saying " an `sf` operation", it just doesn't read well. Can we just say "An operation for..." ?